testsuite/tests/conftest.py

```diff
@@ -122,104 +122,78 @@ def pytest_collection_modifyitems(session, config, items):
-        ]
+    title = _settings["reporting"]["title"] % f"{title} {namespace} {version}"
+
+    metadata.update({
+        "env_for_dynaconf": settings["env_for_dynaconf"],
+        "testsuite-version": testsuite_version,
+        "_3SCALE_TESTS_threescale__admin__url": _settings["threescale"]["admin"]["url"] % "UNKNOWN",
+        "_3SCALE_TESTS_threescale__version": version,
+        "_3SCALE_TESTS_openshift__servers__default__server_url":
+            _settings["openshift"]["servers"]["default"]["server_url"] % "UNKNOWN",
+        "NAMESPACE": namespace,
+        "polarion-testrun-title": title,
+        "polarion-testrun-id": title.replace(".", "_").replace(" ", "_"),
+        "polarion-testrun-status-id": "inprogress",
+        "polarion-project-id": _settings["reporting"]["testsuite_properties"]["polarion_project_id"] % "None",
+        "polarion-response-myteamsname":
+            _settings["reporting"]["testsuite_properties"]["polarion_response_myteamsname"] % "None",
+        "polarion-lookup-method": "name",
+    })
 
     if Capability.OCP4 in CapabilityRegistry():
-        apicast_project = \
-            weakget(settings)["threescale"]["gateway"]["OperatorApicast"]["openshift"]["project_name"] % "None"
-        apicast_version = settings["threescale"]["apicast_operator_version"]
-        catalogsource = weakget(settings)["threescale"]["catalogsource"] % "UNKNOWN"
-        _global_property(config, "openshift-catalogsource", catalogsource)
 
-        header.append(f"testsuite: Apicast operator project = {apicast_project}")
-        header.append(f"testsuite: for Apicast operator version = {apicast_version}")
-        header.append(f"testsuite: catalogsource = {catalogsource}")
+        metadata["_3SCALE_TESTS_threescale__gateway__OperatorApicast__openshift__project_name"] = \
+            _settings["threescale"]["gateway"]["OperatorApicast"]["openshift"]["project_name"] % "None"
+        metadata["_3SCALE_TESTS_threescale__apicast_operator_version"] = \
+            _settings["threescale"]["apicast_operator_version"] % "UNKNOWN"
+        metadata["_3SCALE_TESTS_threescale__catalogsource"] = _settings["threescale"]["catalogsource"] % "UNKNOWN"
+
+    toolboximage = _settings["toolbox"]["podman_image"].split(':')[-1] % "UNKNOWN"
+    toolboxversion = "UNKNOWN"
 
-    if config.getoption("--toolbox"):
-        toolboximage = weakget(settings)["toolbox"]["podman_image"].split(':')[-1] % "UNKNOWN"
-        toolboxversion = "UNKNOWN"
+    if toolboximage != "UNKNOWN":
+        try:
+            toolboxversion = toolbox.run_cmd("-v")['stdout'].strip()
+        except BoxKeyError:
+            warnings.warn("Toolbox executioner is not configured")
 
-        if toolboximage != "UNKNOWN":
-            try:
-                toolboxversion = toolbox.run_cmd("-v")['stdout'].strip()
-            except BoxKeyError:
-                warnings.warn("Toolbox executioner is not configured")
+    metadata["toolbox-version"] = toolboxversion
```

Comment:
I am not sure if we want to include this if toolbox tests aren't run

Reply:
... what breaks ... nothing